web/src/lib/components/shared-components/settings/setting-accordion.svelte

```diff
@@ -19,6 +20,11 @@
       $accordionState = $accordionState;
     }
   };
+
+  onDestroy(() => {
+    $accordionState.delete(key);
+    $accordionState = $accordionState;
```

Comment:
Looks good, you could also do

```suggestion
    setIsOpen(false);
```

Reply:
True, thanks!